### DISCUSSION TITLE
MQTT access  receive

### DISCUSSION THREAD

**Discussion by Ubenga** [omersiar/esp-rfid#561] [Q&A]
Hello, I have a ready MQTT Server, I have a configured topic in a esp-rfid board, I have a rfid reader with a 26 bits wiegand output.
When I start the board with power supply, I receive a MQTT message in the MQTT server, at the topic that I made.
All this is ok, but when I pass a card for te RFID reader, there is no message with any data on the MQTT server at the topic.
What am I doing wrong?
Thanks.

**Comment by matjack1** [comment]
hey @Ubenga can you try with a debug build so that you can get a bit more info on the serial port? If you can do that, please share the logs so that I can give you my opinion. From the info you are sharing, it looks like it's a hardware problem, but difficult to tell without more info, sorry
